src/main/java/jarvis/Formatter.java

```diff
@@ -0,0 +1,193 @@
+package jarvis;
+
+import jarvis.commands.Deadline;
+import jarvis.commands.Event;
+import jarvis.commands.Task;
+import jarvis.commands.UserList;
+import jarvis.display.DisplayMessages;
+import jarvis.exceptions.JarvisException;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Formatter {
+
+    public static String parseUserInput(String[] inputLine, int startIndex, int endIndex) {
+        String[] updatedArray = Arrays.copyOfRange(inputLine, startIndex, endIndex);
+        return String.join(" ", updatedArray);
+
+    }
+
+    public static int indexOf(String[] userInput, String toFind) {
+        // if array is Null
```

Comment:
[Do Not Repeat the Obvious](https://nus-cs2113-ay2122s2.github.io/website/se-book-adapted/chapters/codeQuality.html#do-not-repeat-the-obvious)